.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest ]
-        python-version: [ 3.7, 3.8, 3.9 ]
+        python-version: [ 3.9, 3.11, 3.12 ]
```

Comment:
`3.10` seemed to be interpreted as `3.1` by the CI runner, not sure why. Skipping since I don't think we need the coverage of `3.10` anyways.